packages/web/src/css/fonts.css

```diff
@@ -0,0 +1,20 @@
+@font-face {
+  font-family: "Rajdhani-Bold";
+  src: url("../assets/Fonts/Rajdhani/Rajdhani-Bold.ttf") format("truetype");
```

Comment:
It's not a good idea to use different names for font weights of a same family.
use `font-weight` instead.